presto-native-execution/presto_cpp/main/TaskResource.cpp

```diff
@@ -205,6 +205,9 @@ proxygen::RequestHandler* TaskResource::acknowledgeResults(
           proxygen::ResponseHandler* downstream) {
         try {
           taskManager_.acknowledgeResults(taskId, bufferId, token);
+        } catch (const velox::VeloxException& e) {
+          sendErrorResponse(downstream, e.what());
```

Comment:
What's the difference here?

Reply:
Synced offline

Reply:
Thanks for the explanation. Good to know!